src/controllers/FacetSliderQueryController.ts

```diff
@@ -307,4 +271,85 @@ export class FacetSliderQueryController {
   private getBrowserCompatibleFormat(value: string) {
     return value.replace('@', 'T').replace(/\//g, '-');
   }
+
+  private putGroupByForFullRangeSliderIntoQueryBuilder(queryBuilder: QueryBuilder, basicGroupByRequest: IGroupByRequest) {
+    // We need a group by request for the "full range" that does not take into account the current query (query override)
+    // The goal is to obtain a query wich return the whole range of results without taking into account what the users typed in the search box,
+    // so that the X range of the slider is static between each queries.
+
+    this.groupByRequestForFullRange = queryBuilder.groupByRequests.length;
+    const groupByRequestForFullRange = clone(basicGroupByRequest);
```

Comment:
Those 2 variables with the same name, one being a `number`, the other a `IGroupByRequest` are a little confusing

Reply:
Renamed to `lastGroupByRequestForFullRangeIndex`